src/Tools/dotnet-format/CodeFormatter.cs

```diff
@@ -115,7 +115,7 @@ private static async Task<int> FormatFilesInWorkspaceAsync(ILogger logger, Works
                 logger.LogInformation(Resources.Formatting_code_files_in_project_0, project.Name);
 
                 var formattedSolution = await FormatFilesInProjectAsync(logger, project, codingConventionsManager, optionsApplier, cancellationToken).ConfigureAwait(false);
-                if (!workspace.TryApplyChanges(formattedSolution))
+                if (saveFormattedFiles && !workspace.TryApplyChanges(formattedSolution))
```

Comment:
Should this show how many files are updated though or something? Usually I'd expect --dry-run for any command to show some sort of an output.

Reply:
Or that's logged in FormatFilesInProjectAsync I guess. I was assuming that was pure but I guess no reason for that.